### PR TITLE
[release-v19] OMSAgent July hotfix + update v19 release

### PR DESCRIPTION
**Pull request by ehashman**
```release-note
Hotfix OMSAgent for July release
```

I haven't done one of these patch updates in a while so hopefully this works.

### CODE REVIEW

File: pluginconfig/pluginconfig-311.yaml [hunk 1]
```diff
@@ -177,16 +177,16 @@ versions:
...
       metricsServer: registry.access.redhat.com/openshift3/ose-metrics-server:v3.11.232
-      azureControllers: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      aroAdmissionController: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      canary: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      etcdBackup: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      metricsBridge: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      startup: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      sync: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
-      tlsProxy: osarpint.azurecr.io/openshift-on-azure/azure:v19.0
+      azureControllers: osarpint.azurecr.io/openshift-on-azure/azure:v19.0.1
```
Comment (mjudeikis) [inline pluginconfig/pluginconfig-311.yaml:182]: We never did this Z versioning here. What changed?

Comment (ehashman) [reply pluginconfig/pluginconfig-311.yaml:182]: We've done this once or twice before - happens when only the sync image content changes but none of the plugin code iirc. See:

- https://github.com/openshift/openshift-azure/releases/tag/v10.1.1
- https://github.com/openshift/openshift-azure/releases/tag/v4.1.1

Comment (jim-minter) [reply pluginconfig/pluginconfig-311.yaml:182]: I also don't understand why this is being done, but will lgtm anyway.

**Pull request merged**